protocols/kad/src/handler.rs

```diff
@@ -480,8 +453,11 @@ impl Handler {
             remote_peer_id,
             next_connec_unique_id: UniqueConnecId(0),
             inbound_substreams: Default::default(),
-            outbound_substreams: Default::default(),
-            num_requested_outbound_streams: 0,
+            outbound_substreams: futures_bounded::FuturesMap::new(
+                Duration::from_secs(30),
```

Comment:
What is the origin of the value 30?

On the rust-libp2p IPFS bootstrap node I see a 95th percentile round-trip time of 400ms. Kademlia requests are tiny and thus I would expect one can send them within one round-trip. Adding a **very** large head-room for the case of a busy connection, how about an outbound timeout of 10 seconds?
```suggestion
                Duration::from_secs(10),
```

Reply:
Sure, happy to change it to 10. I didn't give it much thought initially.